src/main/java/com/minecolonies/compatibility/tinkers/TinkersWeaponProxy.java

```diff
@@ -0,0 +1,32 @@
+package com.minecolonies.compatibility.tinkers;
+
+import net.minecraft.item.ItemStack;
+import org.jetbrains.annotations.NotNull;
+import slimeknights.tconstruct.library.tools.SwordCore;
+import slimeknights.tconstruct.library.utils.ToolHelper;
+
+/**
+ * Class to check if certain tinkers items serve as weapons for the guards.
+ */
+public class TinkersWeaponProxy
+{
+    /**
+     * Check if a certain itemstack is a tinkers weapon.
+     * @param stack the stack to check for.
+     * @return true if so.
+     */
+    public boolean isTinkersWeapon(@NotNull final ItemStack stack)
+    {
+        return false;
+    }
+
+    /**
+     * Calculate the actual attack damage of the tinkers weapon.
+     * @param stack the stack.
+     * @return the attack damage.
+     */
+    public double getAttackDamage(@NotNull final ItemStack stack)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "stack". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1172)